app/src/main/java/edu/uc/zhukv/droneradarmap/MapsActivityCurrentPlace.kt

```diff
@@ -18,8 +18,8 @@ class MapsActivityCurrentPlace(var NameOfCity: TextView? = null) {
     val MIN_TIME: Long = 5000
     val MIN_DISTANCE = 1000f
     val REQUEST_CODE = 101
-
-    var location_provider = LocationManager.GPS_PROVIDER
+    private val PERMISSIONS_REQUEST_ACCESS_FINE_LOCATION: Int = 0
```

Comment:
Moved constant declaration so that it can be accessed in multiple functions. According to 
 : https://kotlinlang.org/docs/coding-conventions.html#idiomatic-use-of-language-features